app/i18n/index.js

```diff
@@ -108,10 +108,26 @@ function loadTranslation(locale) {
     }
 }
 
+let momentLocale = DEFAULT_LOCALE;
```

Comment:
Not sure if test will be effective but one way is to export `momentLocale` and have the test like:
```javascript
import {
    DEFAULT_LOCALE,
    momentLocale,
    getTranslations,
} from './index';

describe('i18n utils', () => {
    test('setMomentLocale', () => {
        // On initial run, if moment is set with invalid locale,
        // it should default to 'en'.
        getTranslations('not-valid');
        expect(momentLocale).toEqual(DEFAULT_LOCALE);

        getTranslations('es');
        expect(momentLocale).toEqual('es');

        getTranslations('ES');
        expect(momentLocale).toEqual('es');

        // If moment has locale other than 'en' and attempted to set with invalid locale,
        // it should fallback to current locale.
        getTranslations('not-valid');
        expect(momentLocale).toEqual('es');
    });
});
```